### DISCUSSION TITLE
Simple email and password sign up Next.js without <Auth /> component ?

### DISCUSSION THREAD

**Discussion by jeanhdev** [supabase/supabase#12016] [Questions]
Hello all !

I'm building a simple Next.js authentication with e-mail and password. I want the user to confirm e-mail after sign-up.
Note: I don't want to use the <Auth /> UI component.

My user is redirected at `http://localhost:3000#accessToken...` and I have no idea what to do at this step.

I cannot find the documentation on what to do after the user clicks on the confirmation email after signup...

Can someone points me to where is the documentation - I cannot find it after hours of searching.

Thank you

**Comment by Collin0810** [comment]
Hey @jeanhdev!

If you follow the steps 'Initialize a Next.js app' and 'Set up a Login component' in these [docs](https://supabase.com/docs/guides/getting-started/tutorials/with-nextjs) and use the supabase.auth.signUp() & supabase.auth.signInWithPassword() methods ([docs](https://supabase.com/docs/reference/javascript/auth-signup)) instead of using the `<Auth/>` component, everything should work. After clicking the link in the email confirmation email, the user should be redirected to your site and automatically be authenticated!
